static/js/notifications.js

```diff
@@ -217,14 +217,19 @@ exports.window_has_focus = function () {
     return window_has_focus;
 };
 
-function in_browser_notify(message, title, content) {
+function in_browser_notify(message, title, content, raw_operators, opts) {
     var notification_html = $(templates.render('notification', {gravatar_url: ui.small_avatar_url(message),
                                                                 title: title,
                                                                 content: content}));
     $('.top-right').notify({
         message: {html: notification_html},
         fadeOut: {enabled: true, delay: 4000}
     }).show();
+    $('.top-right').on('click', function () {
+        ui.change_tab_to('#home');
+        narrow.activate(raw_operators, opts);
+    });
+    setTimeout(function () {$('.top-right').unbind("click");}, 5000);
```

Comment:
And also, if multiple notifications arrive in quick succession, it will cause a bit of lag when clicking on them. I also talked about that in my first commit message, that I later removed. But I don't expect this to be a big problem, because the Zulip servers usually aren't being spammed with messages every second.